### ISSUE TITLE
ssl error while launching blazor app in chrome

### ISSUE THREAD

**Issue by jinishans** [dotnet/core#8550]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/aspnet/blazor-tutorial/run
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.
![Uploading Blazor_Error_SSL.jpg…]()

**Comment by adamsitnik** [comment]
Hi @jinishans 

The attachment does not work in both Edge (Chromium based):

![image](https://github.com/dotnet/core/assets/6011991/b9c18e9c-9931-49ea-852f-0aca5988799e)

And Firefox:

![image](https://github.com/dotnet/core/assets/6011991/2ef87773-e91b-45d7-bce4-5602120d1163)

Could you please re-upload the screen shot? Or even better: copy paste the error output/log here?

**Comment by mairaw** [comment]
If you launch the application again, do you still see the same error? 

/cc @danroth27

**Comment by danroth27** [comment]
It seems that browsers made a somewhat recent change that you need to shut down and restart the browser for the certificate trust to get picked up. So it's really the browser that needs to be restarted, not necessarily the app.

**Comment by jinishans** [comment]
Ok sure.

I tried with different browsers still got error.

Will try it again.

**Comment by jinishans** [comment]
It's working now in chrome, not sure how. Tx. 

I was trying to setup optane CMS in parallel, not sure it caused this issue. Will progress more in the tutorial and see how it goes. Tx anyways for your quick response in addressing the issue.

**Comment by adamsitnik** [comment]
@jinishans thanks for letting us know! Please open a new issue if you hit any more troubles.
